Test/git-issues/git-issue-351.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+
+function method DecodeRecursively(s: seq<int>): (b: seq<int>) {
+    seq(|s|, i requires 0 <= i < |s| =>
+        var d := s[0]; // this line produces the error below
```

Comment:
The comment says there's an error, but that before the bug fix. Please reword comment.